src/components/MerchantsPage/MerchantsPage.tsx

```diff
@@ -32,8 +32,8 @@ const MerchantsPage: React.FC<{}> = () => {
         <NavBar />
 
         <div className={styles.merchantsContainer}>
-          {sellers.map((store: any) => (
-            <MerchantCard storeInfo={store} />
+          {sellers.map((store: any, idx: any) => (
```

Comment:
using the index isn't the best for assigning keys for child components because react will confuse children elements whenever it changes dynamically and might create some sorting issues down the lane. More in depth explanation: https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318

Reply:
Good callout

Reply:
good call! fixed throughout :)